v5/v5_with_library/v5_stable.ino

```diff
@@ -1473,10 +1522,24 @@ void inputManager(String command, int inputFrom) {
         command.substring(command.indexOf("{") + 1, command.indexOf("}"));
     sendSMS(strSms, strNumber);
     inputFrom == 3 ? command += "<executed>" : "";
+  } else if (isIn(command, "say to orange pi", "say to orangepi")) {
+    toOrangePi(command.substring(command.indexOf("say to orange") + 16, -1));
+  } else if (isIn(command, "test package renew")) {
+    String _message_ = "Your Offer with 0 Onnet Mins 0 Other Network Mins 0 "
+                       "MBs 10000 SMS with 30day validity ";
+    if (newPackageSubscribed(_message_)) {
+      updatePackageSubscribedDate();
+    }
+  } else if (isIn(command, "test orange pi logs")) {
+    saveItOrangePi(command.substring(command.indexOf("test orange") + 19, -1));
+  } else if (command.indexOf("switch") != -1 &&
+             (command.indexOf("off") != -1 || command.indexOf("on") != -1)) {
+    relayControllerManager(command);
+    inputFrom == 3 ? command += "<executed>" : "";
   } else if ((command.indexOf("check sms sending") != -1) ||
              (command.indexOf("sms sending?") != -1)) {
-    //
-
+    sms_allowed = hasPackage();
+    println("SMS sending is " + sms_allowed ? "allowed!" : "not allowed!");
```

Comment:
I think this line won't work can you please check it if it working fine then please attach a screen snap @mhamidjamil

Reply:
Yes, you are right I got ![image](https://github.com/mhamidjamil/TTGO_TCall/assets/55445517/d6422a60-8867-4715-96f7-b8d8f422c81d), instead of `SMS sending is allowed!`, I have corrected it.